packages/py-api/py_api/services/mailer.py

```diff
@@ -1,22 +1,25 @@
 from email.mime.multipart import MIMEMultipart
 from email.mime.text import MIMEText
-from os import getenv
-from smtplib import SMTP_SSL
+from os import environ
+
+from aiosmtplib import SMTP
 
 
 async def send_mail(receiver: str, subject: str, html: str) -> None:
     port = 465  # SSL
-    email = getenv('HUB_MAIL', '')
-    password = getenv('HUB_PSW', '')
+    email = environ["HUB_MAIL"]
```

Comment:
Don't we have an environment.py file or something of the sorts to load environment variables?